tests/zkevm/test_worst_compute.py

```diff
@@ -282,3 +283,135 @@ def code_loop_precompile_call(calldata: Bytecode, attack_block: Bytecode):
         raise ValueError(f"Code size {len(code)} exceeds maximum code size {MAX_CODE_SIZE}")
 
     return code
+
+
+@pytest.mark.zkevm
+@pytest.mark.valid_from("Cancun")
+@pytest.mark.slow
+@pytest.mark.parametrize(
+    "gas_limit",
+    [36_000_000],
+)
+def test_worst_jumps(
+    blockchain_test: BlockchainTestFiller,
+    pre: Alloc,
+    fork: Fork,
+    gas_limit: int,
+):
+    """Test running a JUMP-intensive contract."""
+    env = Environment(gas_limit=gas_limit)
```

Comment:
```suggestion
    env = Environment()
```
Since the testing-framework added support for `--block-gas-limit` we will parametrize using that. For example, [the CI filler](https://github.com/ethereum/execution-spec-tests/blob/main/.github/configs/feature.yaml#L16) already does this.

If we want to generate the suite for a higher gas limit, we can change the value in the CLI.